### PR TITLE
Fixed issue with back to sign in not working

### PR DESCRIPTION
**Pull request by ErikCH**
#### Description of changes

Back to sign in on force new password page is not working

#### Issue #, if available
#1305 

#### Description of how you validated changes
Added cypress test, and fixed tests in `sign-in-force-new-password-feature` that were all skipped.

#### Checklist

- [x] PR description included
- [x] `yarn test` passes
- [x] Tests are updated
- [x] Relevant documentation is changed or added (and PR referenced) - non needed

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.

### CODE REVIEW

File: .changeset/fuzzy-horses-heal.md [hunk 1]
```diff
@@ -0,0 +1,5 @@
+---
+'@aws-amplify/ui': major
```
Comment (wlee221) [inline .changeset/fuzzy-horses-heal.md:2]: Probably `patch`, right?

File: packages/e2e/features/ui/components/authenticator/sign-in-force-new-password.feature [hunk 2]
```diff
@@ -6,22 +6,35 @@ Feature: Sign In with Force New Password flow
...
 		Then I should see the Force Change Password screen
 
-  @angular @react @vue @skip
+  @angular @react @vue 
   Scenario: User is in a FORCE_CHANGE_PASSWORD state and then enters an invalid new password
     When I select my country code with status "FORCE_CHANGE_PASSWORD"
     And I type my "phone number" with status "FORCE_CHANGE_PASSWORD"
     And I type my password
     And I click the "Sign in" button
+		Then I should see the Force Change Password screen
```
Comment (wlee221) [inline packages/e2e/features/ui/components/authenticator/sign-in-force-new-password.feature:36]: I think there is some spacing problem here

File: packages/e2e/features/ui/components/authenticator/sign-in-force-new-password.feature [hunk 3]
```diff
@@ -6,22 +6,35 @@ Feature: Sign In with Force New Password flow
...
+  @angular @react @vue 
   Scenario: Sign in using a valid phone number and password and user is in a FORCE_CHANGE_PASSWORD state
     When I select my country code with status "FORCE_CHANGE_PASSWORD"
     And I type my "phone number" with status "FORCE_CHANGE_PASSWORD"
     And I type my password
     And I click the "Sign in" button
 		Then I should see the Force Change Password screen
 
-  @angular @react @vue @skip
+  @angular @react @vue
```
Comment (wlee221) [inline packages/e2e/features/ui/components/authenticator/sign-in-force-new-password.feature:30]: Tfw when we remove `@skip` 😄 , thanks!

**Review by wlee221 — COMMENTED** [review]
The actual logic looks good, just some random minor nits!

File: packages/e2e/features/ui/components/authenticator/sign-in-force-new-password.feature (hunk 2 above)
Comment (ErikCH) [reply packages/e2e/features/ui/components/authenticator/sign-in-force-new-password.feature:36]: Will do!

File: packages/e2e/features/ui/components/authenticator/sign-in-force-new-password.feature (hunk 3 above)
Comment (ErikCH) [reply packages/e2e/features/ui/components/authenticator/sign-in-force-new-password.feature:30]: 👍

File: .changeset/fuzzy-horses-heal.md (hunk 1 above)
Comment (ErikCH) [reply .changeset/fuzzy-horses-heal.md:2]: Will do!

**Review by wlee221 — APPROVED** [review]

File: packages/e2e/features/ui/components/authenticator/sign-in-force-new-password.feature [hunk 4]
```diff
@@ -6,22 +6,35 @@ Feature: Sign In with Force New Password flow
 
   Background:
     Given I'm running the example "ui/components/authenticator/sign-in-with-phone"
+    Given I intercept '{ "headers": { "X-Amz-Target": "AWSCognitoIdentityProviderService.RespondToAuthChallenge" } }' with fixture "force-change-password"
```
Comment (ericclemmons) [inline packages/e2e/features/ui/components/authenticator/sign-in-force-new-password.feature:9]: Glad to see these mocks are still working!

**Pull request merged**